profiles/rock3b/rock3b.yaml

```diff
@@ -0,0 +1,9 @@
+arch: arm64
+platform: metal
+secureboot: false
+output:
+  kind: image
+  outFormat: .xz
+  imageOptions:
+    diskSize: 1306525696
+    diskFormat: raw
```

Comment:
https://github.com/siderolabs/sbc-rockchip/blob/main/profiles/helios64/helios64.yaml#L10 this needs to be set